src/hooks/services/markers/useGetMarkers.ts

```diff
@@ -0,0 +1,20 @@
+import { getWarehouseManagementApi } from '@/services/generated-api'
+import { useSuspenseQuery } from '@tanstack/react-query'
+
+//TODO: Когато БЕ оправи pagination да го оправя.
+//Според мен не ни е необходим, защото така си работи перфектно. Да обсъдя с Крис.
```

Comment:
lets leave this as what you have done here since there wont be many markers. also delete the comments.

Reply:
fixed